core/shared/src/test/scala/fs2/Fs2Suite.scala

```diff
@@ -0,0 +1,87 @@
+package fs2
+
+import scala.concurrent.ExecutionContext
+
+import cats.effect.{ContextShift, IO, Sync, Timer}
+import cats.implicits._
+import munit.{Location, ScalaCheckSuite}
+import org.typelevel.discipline.Laws
+
+abstract class Fs2Suite
+    extends ScalaCheckSuite
+    with AsyncPropertySupport
+    with TestPlatform
+    with Generators {
+
+  override def scalaCheckTestParameters =
+    super.scalaCheckTestParameters
+      .withMinSuccessfulTests(if (isJVM) 25 else 5)
+      .withWorkers(1)
+
+  override def munitFlakyOK = true
+
+  implicit val realExecutionContext: ExecutionContext =
```

Comment:
Does this need to be `implicit`? Regardless, the assigned value can be `ExecutionContext.global` instead.

Reply:
Nope, made it non explicit and assigned to `ExecutionContext.global`